tests/Entity/OrderTest.php

```diff
@@ -422,4 +471,91 @@ public function orderTypeProvider(): array
             ['Urine', null, '["1UR10"]']
         ];
     }
+
+    public function getBiobankChangesDataProvider(): array
+    {
+        $ts = new \DateTime('2022-01-03 08:00:00');
+        return [
+            [
+                [
+                    'collectedTs' => null,
+                    'processedTs' => null,
+                ],
+                '{"collected":{"time":1641024000,"user":null,"site":"test","samples":["1SS08","1PS08","1HEP4","1ED04","1ED10","1CFD9","1PXR2","1UR10"]},"processed":{"time":1641024000,"user":null,"site":"test","samples":["1SS08","1PS08"],"samples_ts":{"1SS08":1641024000,"1PS08":1641024000},"centrifuge_type":"swinging_bucket"},"finalized":{"time":1641110400,"site":"test","user":null,"notes":"","samples":["1SS08","1PS08","1HEP4","1ED04","1ED10","1CFD9","1PXR2","1UR10"]}}'
```

Comment:
It looks like these JSON objects are mostly the same.  Would it make sense to have a private method that generated these objects with the expected customizations?

Reply:
As discussed in our standup we'll keep this way since it looks cleaner and easier to read.